### PR TITLE
Bpk 3000 use new colours

### PR DESCRIPTION
**Pull request by tiagoengel**
+ [ ] Check this if you have read and followed the [contributing guidelines](https://github.com/Skyscanner/backpack-android/blob/master/CONTRIBUTING.md)

Remember to include the following changes:
+ [x] `UNRELEASED.md`
+ [x] `README.md`
+ [x] Tests
+ [x] Docs (either update [backpack-docs](https://github.com/Skyscanner/backpack-docs) now, or create a follow up ticket)

_If you are curious about how we review, please read through the [code review guidelines](https://github.com/Skyscanner/backpack/blob/master/CODE_REVIEW_GUIDELINES.md)_

### CODE REVIEW

File: app/src/main/java/net/skyscanner/backpack/demo/data/ExampleBpkCalendarController.kt [hunk 1]
```diff
@@ -74,9 +74,9 @@ internal fun multiColoredExampleCalendarColoring(
   }
   return CalendarColoring(
     setOf(
-      ColoredBucket(ContextCompat.getColor(context, R.color.bpkRed200), redSet),
-      ColoredBucket(ContextCompat.getColor(context, R.color.bpkYellow100), yellowSet),
-      ColoredBucket(ContextCompat.getColor(context, R.color.bpkGreen100), greenSet),
+      ColoredBucket(ContextCompat.getColor(context, R.color.bpkNara), redSet),
```
Comment (bvitaliyg) [inline app/src/main/java/net/skyscanner/backpack/demo/data/ExampleBpkCalendarController.kt]: Let's use azureMonteverde, azureKolkata, azurePanjin here.

File: Backpack/src/androidTest/java/net/skyscanner/backpack/util/BpkThemeTest.kt [hunk 2]
```diff
@@ -33,7 +33,7 @@ class BpkThemeTest {
 
   @Test
   fun test_primary_color() {
-    Assert.assertEquals("default", blue500, BpkTheme.getPrimaryColor(activity))
+    Assert.assertEquals("default", skyBlue, BpkTheme.getPrimaryColor(activity))
```
Comment (bvitaliyg) [inline Backpack/src/androidTest/java/net/skyscanner/backpack/util/BpkThemeTest.kt]: There's some inconsistency in checkbox style. We need to verify it and select only one option.

File: Backpack/src/androidTest/java/net/skyscanner/backpack/util/BpkThemeTest.kt [hunk 3]
```diff
@@ -49,7 +49,7 @@ class BpkThemeTest {
 
     val newContext = BpkTheme.wrapContextWithDefaults(activity)
     newContext.theme.resolveAttribute(R.attr.bpkPrimaryColor, t, true)
-    Assert.assertEquals(blue500, t.data)
+    Assert.assertEquals(skyBlue, t.data)
```
Comment (bvitaliyg) [inline Backpack/src/androidTest/java/net/skyscanner/backpack/util/BpkThemeTest.kt]: Calendar theme attributes (calendarDateSelectedBackgroundColor) etc need to be updated as well.

File: Backpack/src/androidTest/java/net/skyscanner/backpack/util/BpkThemeTest.kt [hunk 4]
```diff
@@ -67,7 +67,7 @@ class BpkThemeTest {
 
     BpkTheme.applyDefaultsToContext(testContext)
     testContext.theme.resolveAttribute(R.attr.bpkPrimaryColor, t, true)
-    Assert.assertEquals(blue500, t.data)
+    Assert.assertEquals(skyBlue, t.data)
```
Comment (bvitaliyg) [inline Backpack/src/androidTest/java/net/skyscanner/backpack/util/BpkThemeTest.kt]: BpkTheme.getPrimaryColor docs refer to the wrong colour now.

File: Backpack/src/androidTest/java/net/skyscanner/backpack/spinner/BpkSpinnerTest.kt [hunk 5]
```diff
@@ -28,7 +28,7 @@ class BpkSpinnerTest {
   fun test_default() {
     Assert.assertFalse(subject.small)
     Assert.assertEquals(BpkSpinner.Type.PRIMARY, subject.type)
-    Assert.assertEquals(getColor(subject, R.color.bpkBlue500), subject.getColor())
+    Assert.assertEquals(getColor(subject, R.color.bpkSkyBlue), subject.getColor())
```
Comment (bvitaliyg) [inline Backpack/src/androidTest/java/net/skyscanner/backpack/spinner/BpkSpinnerTest.kt]: We probably need to check and update BpkResources

File: Backpack/src/androidTest/java/net/skyscanner/backpack/util/BpkThemeTest.kt [hunk 6]
```diff
@@ -23,7 +23,7 @@ class BpkThemeTest {
   internal var activityRule: ActivityTestRule<TestActivity> =
     ActivityTestRule(TestActivity::class.java)
 
-  private val blue500 by unsafeLazy { ContextCompat.getColor(activity, R.color.bpkBlue500) }
+  private val skyBlue by unsafeLazy { ContextCompat.getColor(activity, R.color.bpkSkyBlue) }
```
Comment (bvitaliyg) [inline Backpack/src/androidTest/java/net/skyscanner/backpack/util/BpkThemeTest.kt]: Bpk.chip style refers to wrong colours now.

File: Backpack/src/main/java/net/skyscanner/backpack/badge/BpkBadge.kt [hunk 7]
```diff
@@ -34,15 +34,15 @@ open class BpkBadge @JvmOverloads constructor(
     /**
      * Style for badges with positive messages
      */
-    Success(1, R.color.bpkGreen500, R.color.bpkGray700),
+    Success(1, R.color.bpkMonteverde, R.color.bpkGray700),
```
Comment (bvitaliyg) [inline Backpack/src/main/java/net/skyscanner/backpack/badge/BpkBadge.kt]: HorizontalNavTests.screenshotHorizontalNav_alternate need to have the background colour updated.

File: Backpack/src/main/java/net/skyscanner/backpack/badge/BpkBadge.kt [hunk 8]
```diff
@@ -34,15 +34,15 @@ open class BpkBadge @JvmOverloads constructor(
...
     /**
      *  Style for badges with warning messages
      */
-    Warning(2, R.color.bpkYellow500, R.color.bpkGray700),
+    Warning(2, R.color.bpkErfoud, R.color.bpkGray700),
```
Comment (bvitaliyg) [inline Backpack/src/main/java/net/skyscanner/backpack/badge/BpkBadge.kt]: fragment_text_drawables uses old green

File: app/src/main/java/net/skyscanner/backpack/demo/data/ExampleBpkCalendarController.kt [hunk 9]
```diff
@@ -74,9 +74,9 @@ internal fun multiColoredExampleCalendarColoring(
...
+      ColoredBucket(ContextCompat.getColor(context, R.color.bpkBagan), yellowSet),
+      ColoredBucket(ContextCompat.getColor(context, R.color.bpkSegano), greenSet),
       ColoredBucket(ContextCompat.getColor(context, R.color.bpkGray50), greySet),
       ColoredBucket(null, emptySet, ContextCompat.getColor(context, R.color.bpkPink500))
```
Comment (bvitaliyg) [inline app/src/main/java/net/skyscanner/backpack/demo/data/ExampleBpkCalendarController.kt]: I think we need to use some of the new pinks here.

File: app/src/main/java/net/skyscanner/backpack/demo/data/ExampleBpkCalendarController.kt [hunk 10]
```diff
@@ -74,9 +74,9 @@ internal fun multiColoredExampleCalendarColoring(
   }
   return CalendarColoring(
     setOf(
-      ColoredBucket(ContextCompat.getColor(context, R.color.bpkRed200), redSet),
-      ColoredBucket(ContextCompat.getColor(context, R.color.bpkYellow100), yellowSet),
-      ColoredBucket(ContextCompat.getColor(context, R.color.bpkGreen100), greenSet),
+      ColoredBucket(ContextCompat.getColor(context, R.color.bpkNara), redSet),
+      ColoredBucket(ContextCompat.getColor(context, R.color.bpkBagan), yellowSet),
```
Comment (bvitaliyg) [inline app/src/main/java/net/skyscanner/backpack/demo/data/ExampleBpkCalendarController.kt]: BpkChipTest.screenshotTestCustomSelectedBackground and screenshotTestCustomBackground use old colour

File: app/src/main/java/net/skyscanner/backpack/demo/data/ExampleBpkCalendarController.kt (hunk 1 above)
Comment (tiagoengel) [reply app/src/main/java/net/skyscanner/backpack/demo/data/ExampleBpkCalendarController.kt]: those are not the right colours see https://app.zeplin.io/project/5d6fa72570db2f18bdf8e209/screen/5d6fb58fc1b83b981a512b5e

File: Backpack/src/androidTest/java/net/skyscanner/backpack/util/BpkThemeTest.kt (hunk 2 above)
Comment (tiagoengel) [reply Backpack/src/androidTest/java/net/skyscanner/backpack/util/BpkThemeTest.kt]: done

File: Backpack/src/androidTest/java/net/skyscanner/backpack/util/BpkThemeTest.kt (hunk 3 above)
Comment (tiagoengel) [reply Backpack/src/androidTest/java/net/skyscanner/backpack/util/BpkThemeTest.kt]: done

File: Backpack/src/androidTest/java/net/skyscanner/backpack/util/BpkThemeTest.kt (hunk 4 above)
Comment (tiagoengel) [reply Backpack/src/androidTest/java/net/skyscanner/backpack/util/BpkThemeTest.kt]: done

File: Backpack/src/main/java/net/skyscanner/backpack/badge/BpkBadge.kt (hunk 7 above)
Comment (tiagoengel) [reply Backpack/src/main/java/net/skyscanner/backpack/badge/BpkBadge.kt]: done

File: Backpack/src/main/java/net/skyscanner/backpack/badge/BpkBadge.kt (hunk 8 above)
Comment (tiagoengel) [reply Backpack/src/main/java/net/skyscanner/backpack/badge/BpkBadge.kt]: done

File: app/src/main/java/net/skyscanner/backpack/demo/data/ExampleBpkCalendarController.kt (hunk 9 above)
Comment (tiagoengel) [reply app/src/main/java/net/skyscanner/backpack/demo/data/ExampleBpkCalendarController.kt]: Updated according to this https://app.zeplin.io/project/5d6fa72570db2f18bdf8e209/screen/5d6fb58fc1b83b981a512b5e

File: Backpack/src/androidTest/java/net/skyscanner/backpack/spinner/BpkSpinnerTest.kt (hunk 5 above)
Comment (tiagoengel) [reply Backpack/src/androidTest/java/net/skyscanner/backpack/spinner/BpkSpinnerTest.kt]: done

File: Backpack/src/androidTest/java/net/skyscanner/backpack/util/BpkThemeTest.kt (hunk 6 above)
Comment (tiagoengel) [reply Backpack/src/androidTest/java/net/skyscanner/backpack/util/BpkThemeTest.kt]: removed

**Pull request merged**